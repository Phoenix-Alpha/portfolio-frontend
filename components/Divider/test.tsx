import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Divider } from '.'

describe('<Divider />', () => {
  it('should renders correct <Divider />', () => {
    const wrapper = shallow(<Divider />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
