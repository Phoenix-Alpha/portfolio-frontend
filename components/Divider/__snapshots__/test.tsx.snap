// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should renders correct <Divider /> 1`] = `
<Fragment>
  <div
    className="h-px flex items-center my-1 grow w-full"
  />
</Fragment>
`;
